docs/developer-guide/backend.md

```diff
@@ -50,7 +50,7 @@ To reverse migrations for an app, specify the number of the migration you want t
 
 - `003_add_cat_gif_image_field`
 - `004_tweak_streamfield_options`
-- `005_add_fks_to_insights_pages`
+- `005_add_fks_to_Stories_pages`
```

Comment:
if I understood the above comment correctly, the migration file follows the actual model , so if the add/edit a field called stories , then the migration will be something add_story_..... and since the model has changed here then the migration file name will reflect the new change. And yes all naming must be lowercase